client-app/pages/company/info.vue

```diff
@@ -29,12 +27,49 @@
             </template>
           </VcInput>
         </div>
+      </VcWidget>
+      <VcWidget
+        v-if="userCanEditOrganization || !loadingOrganization || !loadingUser"
+        :title="$t('pages.company.info.labels.company_logo')"
+        size="lg"
+      >
+        <div class="flex gap-4">
+          <div v-if="logoUrl" class="flex flex-col justify-between">
+            <div>
+              <VcImage :alt="$t('pages.company.info.labels.company_logo')" :src="logoUrl" class="logo-preview__image" />
+            </div>
+
+            <VcButton
+              :disabled="!logoUrl"
+              :loading="loadingOrganizationLogo"
+              class="whitespace-nowrap"
+              size="sm"
+              variant="outline"
+              @click="saveOrganizationLogo"
+            >
+              {{ $t("common.buttons.save") }}
+            </VcButton>
+          </div>
```

Comment:
Add empty lines to visually separate tags.

Reply:
fixed